pkg/pdfcpu/types/layout.go

```diff
@@ -404,7 +404,8 @@ func ParsePageFormat(v string) (*Dim, string, error) {
 
 	dim := Dim{d.Width, d.Height}
 	if (d.Portrait() && !portrait) || (d.Landscape() && portrait) {
-		dim.Width, dim.Height = dim.Height, dim.Width
```

Comment:
I found this was causing an error because `Width` and `Height` were getting swapped on `PaperSize[v]`. I forget what the error was though

Reply:
This is kind of dangerous because it is used by other commands.
I will have to check this.

Reply:
So the original code should be fine for cases where you want to enforce portrait or landscape mode by appending P or L respectively to the papersize name.

Reply:
ah - cool. that makes sense. I think I had a broken version of this function in my own code (without the dim copy) and mistakenly edited this one too. Will remove